src/safeposix/syscalls/net_calls.rs

```diff
@@ -1966,6 +2005,8 @@ impl Cage {
                                 let sock_tmp = sockfdobj.handle.clone();
```

Comment:
Can you add some comments about what each socket domain is (AF_UNIX, AF_INET, etc), and what are we trying to achieve inside each of the match blocks?